src/Components/ADT3DSceneBuilder/Internal/VisualRuleForm/VisualRuleForm.tsx

```diff
@@ -258,6 +283,24 @@ const VisualRuleForm: React.FC<IVisualRuleFormProps> = (props) => {
         [visualRuleFormDispatch]
     );
 
+    const handleSaveCondition = useCallback(
```

Comment:
is it indented that we can create a rule without condition? The save button is getting enabled without me adding any condition.
![image](https://user-images.githubusercontent.com/45217314/198155342-3630ba5a-1340-421e-8d35-2f9b99b39b74.png)